src/osd/PG.cc

```diff
@@ -3420,8 +3421,10 @@ void PG::reg_next_scrub()
 
 void PG::unreg_next_scrub()
 {
-  if (is_primary())
+  if (is_primary()) {
     osd->unreg_pg_scrub(info.pgid, scrubber.scrub_reg_stamp);
+    scrubber.scrub_reg_stamp = utime_t();
```

Comment:
@robbat2 I still see the following at top of master for the utime_t ctor:

utime_t() { tv.tv_sec = 0; tv.tv_nsec = 0; }